src/Entity.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Me\BjoernBuettner;
+
+use ArrayAccess;
+use BadMethodCallException;
+use InvalidArgumentException;
+use Iterator;
+use JsonSerializable;
+use Stringable;
+
+abstract class Entity implements JsonSerializable, Stringable, ArrayAccess, Iterator
+{
+    private int $position = 0;
+    private array $properties = [];
+    public function __construct(private ?int $aid)
+    {
+        foreach (get_object_vars($this) as $property => $value) {
+            if (!in_array($property, ['position', 'properties'], true)) {
+                $this->properties[] = $property;
+            }
+        }
+    }
+    final public function getAid(): ?int
+    {
+        return $this->aid;
+    }
+    final public function offsetExists(mixed $offset): bool
```

Comment:
missing blank lines between function end and function start, no consistency in file